src/PaginationLink.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  'aria-label': PropTypes.string,
+  children: PropTypes.node,
+  className: PropTypes.string,
+  next: PropTypes.bool,
+  previous: PropTypes.bool,
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+};
+
+const defaultProps = {
+  tag: 'a',
+};
+
+const PaginationLink = (props) => {
+  const {
+    children,
+    className,
+    next,
+    previous,
+    tag: Tag,
+    ...attributes,
+  } = props;
+
+  const classes = classNames(
+    className,
+    'page-link'
+  );
+
+  let defaultAriaLabel;
+  if (previous) {
+    defaultAriaLabel = 'Previous';
+  } else if (next) {
+    defaultAriaLabel = 'Next';
+  }
+  const ariaLabel = props['aria-label'] || defaultAriaLabel;
+
+  let defaultCaret;
+  if (previous) {
+    defaultCaret = <span>&laquo;</span>;
```

Comment:
I don't think these extra spans are needed anymore since it's taken care of with `<span aria-hidden="true">{children || defaultCaret}</span>`.